app.py

```diff
@@ -52,5 +56,88 @@ def actualise_cpu_data():
     cpu_service.calculate_cpu_usage()
 
 
+## *******  HISTORICAL DATA STORAGE AND VIEWING *********
+@app.route('/api/cpu')
+def get_cpu_usage():
+
+    cpu_usage = psutil.cpu_percent()
+    return jsonify(cpu_usage=cpu_usage)
+
+@app.route('/api/memory')
+def get_memory_usage():
+
+    memory_usage = psutil.virtual_memory().percent
+    return jsonify(memory_usage=memory_usage)
+
+@app.route('/api/disk')
+def get_disk_usage():
+
+    disk_usage = psutil.disk_usage('/').percent
+    return jsonify(disk_usage=disk_usage)
+
+@app.route('/api/network')
+def get_network_io():
+
+    network_io = psutil.net_io_counters().bytes_sent + psutil.net_io_counters().bytes_recv
+    return jsonify(network_io=network_io)
+
```

Comment:
Why redefine these endpoints? They're not part of the ticket

Reply:
Okay...